src/models/orderbook.rs

```diff
@@ -1,10 +1,10 @@
 use crate::models::Order;
 use anyhow::Result;
 use ethcontract::web3::types::Address;
-use parking_lot::RwLock;
 use serde::Deserialize;
 use std::collections::HashMap;
 use std::sync::Arc;
+use std::sync::RwLock;
```

Comment:
I think this needs to be `tokio::sync::RwLock` so that it can be used in an `async` context. Sorry I missed that in the first review.

Reply:
Thanks for catching it now. I would have had to correct just in the next PR 🙄